### ISSUE TITLE
Is Sail actually being used?

### ISSUE THREAD

**Issue by rhymes** [thepracticaldev/dev.to#2573]
I noticed that the [sail](https://rubygems.org/gems/sail) is listed in the Gemfile but it's never actually used.

There's still a table:

https://github.com/thepracticaldev/dev.to/blob/a711e2246e7f4ca87afcd247a63d5f0f773836f8/db/schema.rb#L681-L689

but the route is not activated, nor there's any other code that references it.

I've checked the git history and I found these three commits:

1. Sail replaces FlipFlop: https://github.com/thepracticaldev/dev.to/commit/21870e805a5d1e11790bfc356b4c5c4ea817ba98 from https://github.com/thepracticaldev/dev.to/pull/841
1. The previous commit is reverted: https://github.com/thepracticaldev/dev.to/commit/06e0cbabbbf9a7f0517cabb532de3e5ebdd4d66c
1. FlipFlop is removed from the code: https://github.com/thepracticaldev/dev.to/commit/88cd4c13f31a1fd76002529c78835018a71d618f

To recap: currently there are a few references in the code of "Flipflop" though the gem is not present (mainly in skipped tests that should also probably be removed) and there's still the "sail" table in the database in addition to the gem.

If these are remnants from features that are never used I'd be happy to submit a PR to clean it up, but maybe DEV is using the Sail table externally.

**Comment by jessleenyc** [comment]
@rhymes thanks for bringing this up! We had planned to replace FlipFlop with Sail but I vaguely remember something going wrong on deploy and I don't know if we ever fully fixed it. We don't have anything behind flags right now so we haven't looked at it in a while. I'll bring it up with the team but for now we'd definitely welcome a PR fully removing flip flop since we're not using it!

**Comment by rhymes** [comment]
@jessleenyc thanks for the explanation. I'll send a PR to remove the remnants of flipflop and then maybe a separate PR to remove sail in case you decide to proceed in that direction

**Comment by maestromac** [comment]
We are going to keep Sail. We need to verify that it won't crash us again on staging first before fulling utilizing it. thanks for raising this @rhymes !

Resolved with https://github.com/thepracticaldev/dev.to/pull/2585
